### PR TITLE
updates for 2017 analysis

### PR DESCRIPTION
**Pull request by jeongsumin**
Several things are updated which is datasets, HLT, shell script.. etc. for 2017 analysis.
These are first setup for 2017, so some of things might be modified after this PR.

### CODE REVIEW

**Comment by jeongsumin** [comment]
* 'modified some HLT to using unprescaled trigger' -> this commit is about 2016 HLT

File: NanoAODProduction/data/datasets/NanoAOD/2016/MC.RunIISummer16.rareprocess.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ dataset:
         ? /TTZToLLNuNu_M-10_TuneCUETP8M1_13TeV-amcatnlo-pythia8/RunIISummer16NanoAODv5-PUMoriond17_Nano1June2019_102X_mcRun2_asymptotic_v7_ext1-v1/NANOAODSIM
         :   - /store/mc/RunIISummer16NanoAODv5/TTZToLLNuNu_M-10_TuneCUETP8M1_13TeV-amcatnlo-pythia8/NANOAODSIM/PUMoriond17_Nano1June2019_102X_mcRun2_asymptotic_v7_ext1-v1/60000
             - /store/mc/RunIISummer16NanoAODv5/TTZToLLNuNu_M-10_TuneCUETP8M1_13TeV-amcatnlo-pythia8/NANOAODSIM/PUMoriond17_Nano1June2019_102X_mcRun2_asymptotic_v7_ext1-v1/40000
+>>>>>>> upstream/master
```
Comment (jhgoh) [inline NanoAODProduction/data/datasets/NanoAOD/2016/MC.RunIISummer16.rareprocess.yaml]: fix conflict.

File: TopAnalysis/data/grouping/fcncTrilepton_2017.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            #- Run2017.DoubleEG
+            #- Run2017.MuonEG
+    DataMuElEl:
+        title: "Data"
+        modes: ["MuElEl"]
+        cut: "HLT"
+        datasets:
+            - Run2017.SingleMuon
+            - Run2017.SingleElectron
+            #- Run2017.DoubleMuon
```
Comment (jhgoh) [inline TopAnalysis/data/grouping/fcncTrilepton_2017.yaml]: remove this line

File: TopAnalysis/data/grouping/fcncTrilepton_2017.yaml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            - Run2017.DoubleEG
+            #- Run2017.MuonEG
+    DataMuMuMu:
+        title: "Data"
+        modes: ["MuMuMu"]
+        cut: "HLT"
+        datasets:
+            - Run2017.SingleMuon
+            #- Run2017.SingleElectron
```
Comment (jhgoh) [inline TopAnalysis/data/grouping/fcncTrilepton_2017.yaml]: remove this line

**Pull request merged**